### PR TITLE
feat(eventarc): audit log Event receiver sample

### PR DESCRIPTION
**Pull request by muncus**
## Description

Creates a new sample for receiving Audit Log CloudEvents from eventarc

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by muncus** [comment]
The owlbot post-processor failure is unrelated to this PR (underlying error is a formatting issue with an appengine file not touched in this change).

File: eventarc/auditlog_storage_handler/main.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+from cloudevents.http import from_http
+from flask import Flask, request
+from google.events.cloud.audit import LogEntryData
+
+app = Flask(__name__)
+# [END eventarc_audit_cloudevent_server]
+
+
+# [START eventarc_audit_cloudevent_handler]
```
Comment (grayside) [inline eventarc/auditlog_storage_handler/main.py:26]: I had expected something more like a narrow function than a full app. Should we narrow scope?

Comment (grayside) [inline eventarc/auditlog_storage_handler/main.py:26]: We had discussed changing scope on this to another product, should we hold off until that's been confirmed?

**Review by grayside — CHANGES_REQUESTED** [review]
Thank you for putting together this PR. There's a couple aspects of this framed in questions below that we should clarify before proceeding.

**Pull request closed without merging**